packages/strapi-plugin-versioning/__tests__/versioning.test.ts

```diff
@@ -0,0 +1,87 @@
+const { isModelExists, findUid, getModelFromCtx } = require("../src/services/Versioning");
+const { StrapiBuilder: StrapiBuilder2 } = require("strapi-builder");
+
+describe("versioning test", () => {
+  const ctx = {
+    params: {
+      id: 1,
+    },
+    url: "/content-manager/content-types/application::test.test",
+  };
+  test("isModelExists: should return false if model doesn't exists in strapi object", () => {
+    global.strapi = new StrapiBuilder2()
+      .addModels([{
+        fake: {
+          collectionName: "fakeModelCollectionName",
+          uid: "fakeModelCollectionNameUid",
+        },
+      }])
+      .build();
+    expect(isModelExists(ctx)).toBe(false);
```

Comment:
we talked to change function parameter of `isModelExists` to a modelName

Reply:
fixed